src/components/page/page-styles.js

```diff
@@ -48,4 +49,19 @@ const Pagestyles = useDefaultGlobalStyles
 
     `;
 
+/**
+ * @type {typeof deviceSizes}
+ * @return {string}
+ * Media query helper for "styled-components"
+ */
+export const respondTo = Object.keys(deviceSizes).reduce((accumulator, label) => {
+    /** @type {function(...args)} */
+    accumulator[label] = (...args) => css`
+        @media (min-width: ${deviceSizes[label]}px) {
+            ${css(...args)};
```

Comment:
instead of args via spread - let's put one parameter that we can pass

let's cover it in tests as well:

if we pass working / non working code - what expect